src/core/mesh/qgsmeshlayerprofilegenerator.h

```diff
@@ -54,7 +54,9 @@ class CORE_EXPORT QgsMeshLayerProfileResults : public QgsAbstractProfileResults
     double minZ = std::numeric_limits< double >::max();
     double maxZ = std::numeric_limits< double >::lowest();
 
+    Qgis::ProfileSurfaceSymbology symbology = Qgis::ProfileSurfaceSymbology::Line;
     std::unique_ptr< QgsLineSymbol > lineSymbol;
```

Comment:
what is the reasoning for public members (instead of private + setter/getters)?
do you see a way of being able to distinguish them by semantics (like private and static)?

Reply:
Those are all private classes -- it's just a bit quicker to make them public and avoid the getters/setters